#159503755 Add Option For Vendors To Send Newsletters

iverenshaguy opened the pull request:
#### What does this PR do?
This pull request adds the option for vendors to send newsletters to registered users of the app. It also adds two default templates for the vendors to choose from or modify.
#### Description of Task to be completed?
- add newsletter option to email settings
- add default and alternative newsletter templates to email templates
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
We previously thought that the email provider had to be configured manually but we've found out that the vendor/admin configures the email provider through the dashboard.
#### What are the relevant pivotal tracker stories?
#159503755
#### Screenshots or gifs (if appropriate):
<img width="1116" alt="screen shot 2018-08-23 at 9 28 14 pm" src="https://user-images.githubusercontent.com/22734663/44550234-814c0a80-a71b-11e8-94be-e83ad2f7c487.png">
<img width="1440" alt="screen shot 2018-08-23 at 9 27 27 pm" src="https://user-images.githubusercontent.com/22734663/44550235-814c0a80-a71b-11e8-9271-9223f221bd67.png">
<img width="1440" alt="screen shot 2018-08-23 at 9 27 09 pm" src="https://user-images.githubusercontent.com/22734663/44550236-814c0a80-a71b-11e8-9ae8-5f481b89ddeb.png">

#### Questions:
N/A

Stylll reviewed (approved):
Nice Job.

rovilay reviewed (approved):
Good Work. Keep it up.

iverenshaguy committed:
feat(newsletter): add option to send newsletters

- add newsletter option to email settings
- add default and alternative newsletter templates to email templates

[Finishes #159503755]

ozimos reviewed (approved):
Looks good to me. Approved
